tests/test_nuxeo.py

```diff
@@ -7,6 +7,7 @@
 
 import pytest
 import requests
+from _pytest.recwarn import warns
```

Comment:
You should not do that, see https://docs.pytest.org/en/latest/warnings.html#warns how to catch warnings ;)
You just need to do `with pytest.warns(...): ...`